tests/unit/migrationV5.test.ts

```diff
@@ -0,0 +1,112 @@
+import { describe, expect, it } from "vitest";
+
+import { getSdk } from "../../src/shared/utils/cache/get-sdk";
+import { getChain } from "../../src/shared/utils/chain";
+import { thirdwebClient } from "../../src/shared/utils/sdk";
+import { getWalletBalance } from "thirdweb/wallets";
+import { getBalance } from "thirdweb/extensions/erc20";
+import { getContractEvents } from "thirdweb";
+import { getContract as getContractV5 } from "thirdweb";
+import { getContract as getContractV4 } from "../../src/shared/utils/cache/get-contract";
+import { mapEventsV4ToV5 } from "../../src/server/routes/contract/events/get-all-events";
+
+/**
```

Comment:
Hopefully these tests were autogen, thanks for adding them!

Don't need this comment here. The thirdweb secret key should be expected in `.env.test` already.

Reply:
autogen? I write my own code!